app/components/forms/wizard/attendee-step.js

```diff
@@ -12,6 +12,10 @@ export default Component.extend(FormMixin, {
     return this.data.customForms?.filter(field => !field.isFixed);
   }),
 
+  ticketsPresent: computed('data.event.tickets.@each', function() {
+    return this.data.event.tickets.length > 0;
+  }),
+ 
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6112249)